shared/src/test/diff/nu/CommaOperator.mls

```diff
@@ -0,0 +1,388 @@
+:NewDefs
+
+
+1; 2
+//│ 2
+//│ res
+//│     = 1
+//│ res
+//│     = 2
+
+// :dp
+:js
+1, 2
+//│ 2
+//│ // Prelude
+//│ class TypingUnit1 {}
+//│ const typing_unit1 = new TypingUnit1;
+//│ // Query 1
+//│ res = (1 , 2);
+//│ // End of generated code
+//│ res
+//│     = 2
+
+(1, 2)
+//│ 2
+//│ res
+//│     = 2
+
+(1, 2, 3)
+//│ 3
+//│ res
+//│     = 3
+
+1, ()
+//│ ()
+//│ res
+//│     = undefined
+
+log(1), 2
+//│ 2
+//│ res
+//│     = 2
+//│ // Output
+//│ 1
+
+
+(),
+  1
```

Comment:
What if

```
(),
1
```

Reply:
Good question. Currently, `Unexpected newline in expression position`.

Reply:
Just pushed a commit to allow it.